### ISSUE TITLE
EXC_BAD_ACCESS KERN_INVALID_ADDRESS

### ISSUE THREAD

**Issue by ZuzooVn** [lightdesign/LDProgressView#21]
I have this problems with only iOS 8.2

```objc
if ([self.showStroke boolValue]) { // CRASH HERE
    CGContextSetStrokeColorWithColor(context, [[self.color darkerColor] darkerColor].CGColor);
    [roundedRect stroke];
}
```

**Comment by rcdilorenzo** [comment]
Could you provide a specific scenario where this occurs?

**Comment by nicolasgoutaland** [comment]
Hi,
problem is in ***darkerColor*** method. I think you are using an old release of LDProgressView. This problem occurs only on arm64 devices, and was fixed in version 1.2.1 #11 .
Regards
